app/javascript/src/components/Dashboard/Contacts/ContactTable.js

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import { Avatar, Button, Checkbox, Tooltip } from "neetoui";
+
+export default function ContactTable({
+  selectedContactIds,
+  setSelectedContactIds,
+  contacts = [],
+  onContactDelete,
+}) {
+  return (
+    <div className="w-full px-4">
+      <table className="nui-table nui-table--checkbox nui-table--actions">
```

Comment:
Please add classname `nui-table--hover nui-table--actions nui-table--avatar`. 
`nui-table--actions` helps in the hover transition effects for `IconButtons`.